### PR TITLE
refactor(gov): generate page on buildtime

### PR DESCRIPTION
**Pull request by erickzhao**
in preparation for a cron build

### CODE REVIEW

File: src/plugins/governance/fetch-data.ts [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+import toString from 'mdast-util-to-string';
+import { Node, Parent } from 'unist';
+import { Table, Link } from 'mdast';
+import * as dotenv from 'dotenv';
+dotenv.config();
+
+export interface WorkingGroup {
+  name: string;
+  description: string;
+  chair: string;
```
Comment (dsanders11) [inline src/plugins/governance/fetch-data.ts:13]: Should we be forward looking and make `chair` optional?

Comment (erickzhao) [reply src/plugins/governance/fetch-data.ts:11]: yeah, that makes sense.

Comment (erickzhao) [reply src/plugins/governance/fetch-data.ts]: 9600d2e

**Review by georgexu99 — APPROVED** [review]

**Comment by dsanders11** [comment]
@erickzhao, there's a bug in the merged version of this, see the deployment - it renders with `chair &&` visible on the page.

**Pull request merged**